### DISCUSSION TITLE
Embed fontawesome in a generated powerpoint

### DISCUSSION THREAD

**Discussion by Edouard-Legoupil** [davidgohel/officer#474] [Q&A]
Hello - I am using  https://fontawesome.com/icons/categories/humanitarian   within a generated powerpoint - 

When I open it locally it works well as the font is de facto installed on my system - but when I put this pptx online, then it appears that  the option =  "Preserve fidelity when sharing this presentation, select the Embed fonts in the file check box." did not get in.. 

Any ideas on to enable this?

**Comment by davidgohel** [comment]
Hello,

This is an ongoing topic :) We will see if it is possible to embed `*.ttf`. It's not possible to be more precise for the moment because we don't know yet how this feature of 'Office' works. We found an internal MS trick that transforms the font file into mysterious zip file and we don't know if we can get rid of it or reproduce it.

**Reply by Edouard-Legoupil** [reply]
> We found an internal MS trick that transforms the font file into mysterious zip file and we don't know if we can get rid of it or reproduce it. 

Sounds good! Ready to test whenever you have something! -- Was also exploring as a work-around the opportunity to have the font installed by default on our 365 tenants - but that does not seem straightforward either..
